src/main/io/vtx_tramp.c

```diff
@@ -620,29 +581,19 @@ bool vtxTrampInit(void)
 
     // XXX Effect of USE_VTX_COMMON should be reviewed, as following call to vtxInit will do nothing if vtxCommonSetDevice is not called.
 #if defined(USE_VTX_COMMON)
-#if defined(USE_VTX_TABLE)
-    vtxTramp.capability.bandCount = vtxTableBandCount;
-    vtxTramp.capability.channelCount = vtxTableChannelCount;
-    vtxTramp.capability.powerCount = vtxTablePowerLevels;
-    vtxTramp.frequencyTable = (uint16_t *)vtxTableFrequency;
-    vtxTramp.bandNames = vtxTableBandNames;
-    vtxTramp.bandLetters = vtxTableBandLetters;
-    vtxTramp.channelNames = vtxTableChannelNames;
-    vtxTramp.powerNames = vtxTablePowerLabels;
-    vtxTramp.powerValues = vtxTablePowerValues;
-#else
-    vtxTramp.capability.bandCount = VTX_TRAMP_BAND_COUNT;
-    vtxTramp.capability.channelCount = VTX_TRAMP_CHANNEL_COUNT;
-    vtxTramp.capability.powerCount = sizeof(trampPowerTable),
-    vtxTramp.frequencyTable = vtxStringFrequencyTable();
-    vtxTramp.bandNames = vtxStringBandNames();
-    vtxTramp.bandLetters = vtxStringBandLetters();
-    vtxTramp.channelNames = vtxStringChannelNames();
-    vtxTramp.powerNames = trampPowerNames;
-    vtxTramp.powerValues = trampPowerTable;
-#endif
 
     vtxCommonSetDevice(&vtxTramp);
+#ifndef USE_VTX_TABLE
+    //without USE_VTX_TABLE, fill vtxTable variables with default settings (instead of loading them from PG)
+    vtxTablePowerLevels = VTX_TRAMP_POWER_COUNT;
+    for(int i = 0; i < VTX_TRAMP_POWER_COUNT + 1; i++) {
```

Comment:
Space after `for` missing.

Reply:
Alright i grew tired of it and managed to get astyle to run. Hopefully that will help me catch these things before pushing in the future. Thanks for your patience once again.